spec/quality_spec.rb

```diff
@@ -73,6 +89,18 @@ def check_for_extra_spaces(filename)
     expect(error_messages.compact).to be_well_formed
   end
 
+  it "maitains language quality of the documentation" do
+    included = /ronn/
+    error_messages = []
+    Dir.chdir(File.expand_path("../../man", __FILE__)) do
```

Comment:
nah, you can probably just run it over the source files without any parsing required